### DISCUSSION TITLE
v3.1.0-beta1 - CUDA disk-16 - only uses 8 GB RAM

### DISCUSSION THREAD

**Discussion by CityJumperAC** [Chia-Network/bladebit#415] [Q&A]
I am plotting WINdows-CUDA with disk-16 option,

i recognizend that it only uses 8GB RAM, is this the expected behavior?

Will there be a disk-32 option to use at least 16gb of my 32gb RAM?

Greetz
Michael

**Comment by CityJumperAC** [comment]
same on 3.1.0-rc1

**Comment by harold-b** [comment]
This is normal and expected. Windows requires at least double the RAM that is needed in the host.

Moving to discussions

**Reply by CityJumperAC** [reply]
Hmm ok,

But -disk128 is using 128gb ram ,

This is confusing
